src/redux/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
```

Comment:
Zrob sobie nowy folder `src/utils` i tam wrzuc ten plik. I teraz masz dwie opcje:
- zmienic nazwe na cos w stylu `stateLocalStorage` bo teraz nazwa mowi ze to sa jakies ogolne utilsy ale tak na prawde masz tam zhardcodowane zapisywanie i wyciaganie `state` wiec nie uzyjesz tego jesli chcialbys zapisac/odczytac inna wartosc z LS
albo
- zrobic to bardziej generyczne, czyli reuzywalne. Mozesz np do loadState przekazywac name, czyli `... loadState = (name) => {...` i pozniej `const serializedState = localStorage.getItem(name)`. I dzieki temu mozesz sobie tych utilsow uzywac do zapisywania czego chcesz.